doc/source/whatsnew/v0.19.0.txt

```diff
@@ -492,6 +492,7 @@ Deprecations
 - ``Categorical.reshape`` has been deprecated and will be removed in a subsequent release (:issue:`12882`)
 - ``Series.reshape`` has been deprecated and will be removed in a subsequent release (:issue:`12882`)
 
+- ``DataFrame.to_sql()`` has deprecated the ``flavor`` parameter (:issue:`13611`)
```

Comment:
Can you emphasize that is is a superfluous keyword (so no reason to use it), since sqlite is the only supported connection when not using sqlalchemy?